reverb/pip_package/setup.py

```diff
@@ -111,6 +111,16 @@ def run_setup(self):
       long_description = f.read()
 
     version, project_name = self._get_version()
+
+    so_lib_paths = [
+        i for i in os.listdir('.')
+        if os.path.isdir(i) and fnmatch.fnmatch(i, '_solib_*')
```

Comment:
use d instead of i for the temp var, and you can just use d.startswith('_solib_') right?

Reply:
copied from https://github.com/tensorflow/tensorflow/blob/v2.4.1/tensorflow/tools/pip_package/setup.py#L258